### DISCUSSION TITLE
Issue with Burp notification on Telegram.

### DISCUSSION THREAD

**Discussion by s-kustm** [projectdiscovery/notify#49] [General]
Hi team,
It's amazing to see the notification on Telegram/Discord when a tool finds a bug. I appreciate your work.
When I run the command on Linux **' echo "Working" | notify'** it sends the notification to both Telegram/Discord, but when I run **'notify'** it sends the notification to Discord only. I am not getting Burp Collabrator interaction on my Telegram whereas I get above on Discord. 

Is this the functionality of **notify** or am I missing something? Please suggest on above.

Best Regards

**Comment by ehsandeep** [comment]
> Telegram notification does not support burp collaborator.

Yes, this is intended.
